kvstore/simplemap/simplemap_test.go

```diff
@@ -0,0 +1,135 @@
+package simplemap
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"fmt"
+	"testing"
+)
+
+func TestSimpleMap(t *testing.T) {
```

Comment:
Do we need this file? Is this not covered in `simplemap/kvstore_test.go`?

Reply:
I kept it because it was part of the former removed SimpleMap, I only adapted it to this version.
But I guess you are right, the kvstore_test should already cover that. I removed the file again.